taskcluster/configs/config.ci.yml

```diff
@@ -61,6 +61,7 @@ datasets:
   train:
     - opus_ada83/v1
     - opus_ELRC-3075-wikipedia_health/v1
+    - url_https://storage.googleapis.com/releng-translations-dev/data/en-ru/pytest-dataset.[LANG].zst
```

Comment:
you need to add this one to `taskcluster/translations_taskgraph/parameters.py` to include in CI

Reply:
This is creating a failure: `requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/translations.cache.level-3.dataset.dataset-url-https_storage_googleapis_com_releng-translations-dev_data_en-ru_pytest-dataset_%5BLANG%5D_zst-ru-en.hash.f454d0533bf5a4b1bb7ac2c20236a45d97d4a19cd09dbc13c24ef09922bbf58d
`

I'll admit I don't know what this is doing exactly.

Reply:
I added this to the quick Russian training config, so we do have some coverage in CI for that side.

Reply:
@bhearsum Do you have any idea what's going on here? I guess `config.ci.yml` is _not_ actually used for the config for CI, it's actually `taskcluster/translations_taskgraph/parameters.py`, and it's failing to get something from the cache: https://github.com/mozilla/firefox-translations-training/pull/420/checks?check_run_id=22100328956

This seems like a Taskcluster-specific issue.

Reply:
The `[` and `]` characters are invalid in a cache URL